clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/view/GoogleClusterProvider.groovy

```diff
@@ -220,7 +221,7 @@ class GoogleClusterProvider implements ClusterProvider<GoogleCluster.View> {
       serverGroupData.each { CacheData serverGroupCacheData ->
         GoogleServerGroup serverGroup = serverGroupFromCacheData(serverGroupCacheData, clusterView.accountName, instances, securityGroups, loadBalancers)
         clusterView.serverGroups << serverGroup.view
-        clusterView.loadBalancers.addAll(serverGroup.loadBalancers*.view)
+        clusterView.loadBalancers.addAll(serverGroup.loadBalancers)
```

Comment:
Referencing nonexistent property `GoogleLoadBalancerView.view`.
Using java, this line causes exception.